### DISCUSSION TITLE
How to regenerate `rust_builder` folder?

### DISCUSSION THREAD

**Discussion by kjxbyz** [fzyzcjy/flutter_rust_bridge#2455] [Q&A] [answered]
>Please ignore this folder, which is just glue to build Rust with Flutter.
>
Does this mean that the folder is not included in the VCS? How can I regenerate this folder in the CI/CD system?

**Comment by fzyzcjy** [comment, marked as answer]
> Does this mean that the folder is not included in the VCS?

It seems it is version checked. Feel free to ping me if not!

**Reply by kjxbyz** [reply]
>Please ignore this folder, which is just glue to build Rust with Flutter.
>
Oh, I misunderstood. I thought this folder could ignore the version check. Thanks!

**Reply by fzyzcjy** [reply]
You are welcome!
